### PR TITLE
adding error to useform incase entering wrong address

### LINKED ISSUE #1204: User-friendly message should be displayed if the user did not enter address,while creating organization
objective:User-friendly message should be displayed if the user did not enter address,while creating a new ORGANIZATION
Expected: Address is required. (as we did for other field for ex: below)

image

Actual: Refer image below

image

### PR DESCRIPTION
**Pull request by OzkanAbdullahoglu**
### What does this pull request aim to achieve? 💯
Resolves #1204 
_Please be concise and link any related issue(s):_

- [x] I have checked that no one else is working on similar changes.
- [x] I have read the latest [**README**](https://github.com/FightPandemics/FightPandemics/blob/master/README.md) and understand the development workflow.
- [x] The name of this branch is: **`feature/<branch_name>`** (need to have cloned the repo not forked).
- [x] This branch is rebased/merged with the **latest master**.
- [x] There are no merge conflicts.
- [x] There are no console warnings and errors.
- [x] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".

### CODE REVIEW

File: client/src/pages/CreateOrganisationProfile.js [hunk 1]
```diff
@@ -147,6 +147,11 @@ const CreateOrgProfile = (props) => {
             type: CREATE_Organisation_ERROR,
             error: `Creating organisation failed, reason: ${message}`,
           });
+          setError(
+            "location",
+            "required",
+            "Please select an address from the drop-down",
```
Comment (soniaroh) [inline client/src/pages/CreateOrganisationProfile.js]: (according to #1204 ) they want the message to be `Address is required` instead, so it is consistent with other fields, e.g. Email is required.

**Comment by nr431207** [comment]
@sheeba1 the error on top of page is still showing, do we also want that to be changed to something more user friendly? or is this ticket solely to address the `Location is required` to be added under location field? 
Screen Shot 2020-07-18 at 1 31 00 PM

**Comment by sheeba1** [comment]
@OzkanAbdullahoglu  @nr431207  yes, the error in red should be user-friendly.

**Comment by nr431207** [comment]
looks good to me! we got rid of that red box error on top though, @sheeba1 can you confirm if this is ok?

**Comment by galeza** [comment]
Hi!

We have a different error messages for Individual Profile and Organisation Profile when Location is empty:
Screen Shot 2020-07-21 at 13 03 43
Screen Shot 2020-07-21 at 13 03 04

**Comment by OzkanAbdullahoglu** [comment]
Because while I fixed the organisation profile I forgot to check individual profile. and It led us this inconsistency, I will handle that.

**Comment by OzkanAbdullahoglu** [comment]
For your information in `EditAccount.js` and `EditOrganisationAccount.js` , the error messages are still like 

> "Please select an address from the drop-down"

. I thought that maybe we should leave them as they are. What is your opinion?

**Comment by sheeba1** [comment]
@OzkanAbdullahoglu Let us have this message in both places.

"Please select an address from the drop-down". In that way there is uniformity for the error message displayed for Address field
Thanks!

**Comment by idjevm** [comment]
I don't think that `Please select an address from the drop-down` is ideal. I think the first time around it should be something like `Address required. Please enter your address and select from the drop-down` or something along those lines.

**Comment by OzkanAbdullahoglu** [comment]
right now it is all set `Please select an address from the drop-down`
but I think to create we can use `Address is required ` in `CreateOrganisationProfile` and `CreateUserProfile`
and to edit we can use `Please select an address from the drop-down` in `EditAccount` and `EditOrganisationalAccount`

**Comment by OzkanAbdullahoglu** [comment]
chrome_2020-07-22_03-24-31
chrome_2020-07-22_03-23-40
@galeza @sheeba1   What do you think about these ones?

**Comment by sheeba1** [comment]
I am fine with above message

**Comment by OzkanAbdullahoglu** [comment]
@galeza are you agree too? if so I will push the latest update then we can merge and close this PR.

**Comment by sheeba1** [comment]
@OzkanAbdullahoglu , While creating an organization I saw only "Please select address from the drop-down".Not the one mentioned for create profile in above picture 



image

**Comment by sheeba1** [comment]
individual profile

image

**Comment by OzkanAbdullahoglu** [comment]
> individual profile
> 
> image

Yeap I know. I haven't updated all. I just showed you how it would look like If we all agree on this message. Then I will implement it to all 4 files which we mentioned before.

**Comment by sheeba1** [comment]
verified for individual and Organization profile creation and editing and received message as expected

"Address is required. Please enter your address and select it from the drop-down"

**Pull request merged**